.changeset/two-bugs-sing.md

```diff
@@ -0,0 +1,5 @@
+---
+'@cloudflare/next-on-pages': patch
+---
+
+Specify the `framework` is `nextjs` in the Vercel `project.json` file.
```

Comment:
does this need a changeset? can user get effected by this in any way?

Reply:
It's altering how the Vercel CLI might interpret a project, so I think a small changeset is appropriate. Previously the Vercel CLI might also detect the Node.js builder for middleware functions because it didn't know for sure that it was a Next.js project, so this alters its behaviour in a tiny way by telling it that it's Next.js

Reply:
mh... I'm a bit hesitant by the fact that we do this under the hood, so it's not really a user facing thing? 🤔 

although not 100% sure about it I'm fine with it 🙂👍